docs/menu.yml

```diff
@@ -153,8 +153,6 @@ catalog:
                   path: /en/setup/plugins/server_grpc-server
                 - name: HTTP Server
                   path: /en/setup/plugins/server_http-server
-                - name: Prometheus Server
-                  path: /en/setup/plugins/server_prometheus-server
```

Comment:
You removed this one, where is the new one?

Reply:
I think we need a `telemetry export` document.

Reply:
Add to a new example.